text/setup.py

```diff
@@ -22,19 +22,19 @@
     'scipy',
     'pandas',
     'scikit-learn',
-    'fairscale',
-    'scikit-image',
-    'boto3',
-    'smart_open',
-    'pytorch_lightning>=1.5.9',
-    'torchmetrics',
-    'transformers>=4.16.2',
     'Pillow',
     'tqdm',
-    'nptyping',
-    'omegaconf',
-    'timm',
-    'sentencepiece',
+    'fairscale<0.4.6',
+    'scikit-image<0.19.3',
+    'boto3<1.21.4',
```

Comment:
Why do we need boto3?

Reply:
I feel that depend on boto3 is fine. It's actually a dependency of autogluon.common. It is used in the test-cases in text.